### PR TITLE
config(migrations) - post migrations output to github PRs

### PR DESCRIPTION
**Pull request by onewland**
This does a diff with master and uses the existing Sentry commenting code to write out any new migration SQL to the current PR

### CODE REVIEW

File: snuba/migrations/groups.py [hunk 1]
```diff
@@ -155,6 +155,7 @@ def get_migrations(self) -> Sequence[str]:
             "0002_outcomes_remove_size_and_bytes",
             "0003_outcomes_add_category_and_quantity",
             "0004_outcomes_matview_additions",
+            "0005_outcomes_copy",
```
Comment (onewland) [inline snuba/migrations/groups.py:158]: This change will be removed before merge (needed for testing)

File: snuba/migrations/snuba_migrations/outcomes/0005_outcomes_copy.py [hunk 2]
```diff
@@ -0,0 +1,128 @@
+from typing import Sequence
```
Comment (onewland) [inline snuba/migrations/snuba_migrations/outcomes/0005_outcomes_copy.py:1]: This file will be removed before merge (needed for testing)

**Comment by lynnagara** [comment]
This is cool! Does it rerun and repost the DDL on every push? IIRC Sentry updates the original message- can we do the same?

**Comment by onewland** [comment]
> This is cool! Does it rerun and repost the DDL on every push? IIRC Sentry updates the original message- can we do the same?

We're reusing their action and so it does reuse the existing message on every push

File: .github/workflows/ddl-changes.yml [hunk 3]
```diff
@@ -0,0 +1,38 @@
...
+          python -m pip install --upgrade pip
+          make install-python-dependencies
+      - name: Run the migration script
+        run: |
+          SNUBA_SETTINGS=test_distributed python scripts/ddl-changes.py
+      - name: Generate SQL for migration
+        uses: getsentry/action-migrations@v1.0.8
+        env:
+          SENTRY_LOG_LEVEL: ERROR
+          PGPASSWORD: postgres
```
Comment (nikhars) [inline .github/workflows/ddl-changes.yml:33]: Is this a security concern? Does Sentry open source also do this?

**Review by nikhars — COMMENTED** [review]
Nice job. Left a followup question.

File: .github/workflows/ddl-changes.yml (hunk 3 above)
Comment (onewland) [reply .github/workflows/ddl-changes.yml:33]: This is copied directly from the way sentry does it, but I actually think I should get rid of SENTRY_LOG_LEVEL and PGPASSWORD as they don't apply to us

**Review by nikhars — APPROVED** [review]
Please remove the migrations from the PR which were added for testing.

**Pull request merged**